src/Application.vala

```diff
@@ -158,6 +182,50 @@ public class AppCenter.App : Granite.Application {
             });
         }
     }
+
+    private void on_operation_finished (AppCenterCore.Package package, AppCenterCore.Package.State operation, Error? error) {
+        switch (operation) {
+            case AppCenterCore.Package.State.INSTALLING:
```

Comment:
It would be good if we could have an un-installed version of this so that when stuff has been un-installed in the background, you get a notification to say it has succeeded (or not).